protos/settlement.proto

```diff
@@ -0,0 +1,45 @@
+// Copyright 2018 Bitwise IO
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// -----------------------------------------------------------------------------
+
+syntax = "proto3";
+
+message Settlement {
+    enum Action {
+        BUY = 0;
+        SELL = 1;
+    }
+
+    // The order_id of the order which is being settled
+    string order_id = 1;
+
+    // The name of the firm on whose behalf the order was entered
+    string ordering_firm = 2;
+
+    // The name of the firm that supplied the quote matched with the order
+    string quoting_firm_id = 3;
+
+    // Buy or sell settlement from the perspective of the ordering firm
+    Action action = 4;
+
+    // The quantity of bonds that are transferred
+    uint64 bond_quantity = 5;
+
+    // The amount of cash that is transferred
+    float currency_amount = 6;
```

Comment:
As per @peterschwarz comment on Slack, I imagine currency amounts should not be floats in the protos either. 

> Currency amount should be an integer, probably millions of a single unit
e.g. $1 == 1,000,000